packages/cog/src/cog/covering.ts

```diff
@@ -1,43 +1,25 @@
-import { GeoJson, QuadKey, QuadKeyTrie, TileCover } from '@basemaps/geo';
+import { QuadKeyTrie, TileCover } from '@basemaps/geo';
 import { JobCutline } from './job.cutline';
 import { SourceMetadata } from './types';
 
 export const Covering = {
     /**
      * Generate an optimized WebMercator tile cover for the supplied polygons
-     * @param featureCollection Polygons in GeoJson WGS84
-     * @param minZoom Minimal zoom level tile to use
-     * @param maxZoom Highest zoom level of tile to use
-     * @param maxTiles Max number of tiles to be a "valid" covering
+     * @param featureCollection Source TIff Polygons in GeoJson WGS84
+
+     * @param cutline A cutline to apply. Can be empty in which case the source Tiff bounds will be
+     * used
      */
-    optimize(sourceMetadata: SourceMetadata, cutline: JobCutline, maxZoom = 13, maxTiles = 25): QuadKeyTrie {
-        const featureCollection = sourceMetadata.bounds;
+    optimize(sourceMetadata: SourceMetadata, cutline: JobCutline): QuadKeyTrie {
+        const sourceZ = sourceMetadata.resolution;
```

Comment:
I think we are not using the word resolution correctly here, this should be something like `zoom`

I think we should use 
`Resolution` for pixels/meter
`Zoom` Tile Zoom level

Reply:
Yep we talked about renaming it. Shall I go ahead and call it `zoom`?

Reply:
Yup just rename it